.gitignore

```diff
@@ -1,6 +1,9 @@
 /openshift-controller-manager
+/openshift-controller-manager-tests-ext
 .vscode/
 .idea/
+*.tmp
+*~
```

Comment:
Do we have these directories `*.tmp, ~`?

Reply:
.openshift-tests-extension/openshift_payload_openshift-controller-manager.json is needed to add.